lib/steps.js

```diff
@@ -935,8 +963,22 @@ After(async function(scenario) {
         });
       } else {
         // Save/download a picture of the screen that's showing during the unexpected status
-        let safe_filename = await scope.getSafeScenarioFilename( scope, { prefix: `error` });
-        await scope.page.screenshot({ path: `${ safe_filename }.jpg`, type: `jpeg`, fullPage: true });
+        // Save one copy in the outer-most artifact folder
+        let scenario_filename = await scope.getSafeScenarioFilename( scope, { prefix: `error_on` });
+        await scope.page.screenshot({
+          path: `${ scope.paths.artifacts }/${ scenario_filename }.jpg`,
+          type: `jpeg`,
+          fullPage: true
+        });
```

Comment:
It's a tiny bit faster, but I like having them in both places. The screenshot should definitely be in the Scenario folder that errored, but having all of them in the main folder lets you scroll through them really easily, getting an idea of a pattern if there is one instead of hopping back and forth between folders. I'm open to more thoughts though. Do you want to make a discussion item?

Reply:
Sorry, I agree with having two screenshots, I was just saying that we don't need to take the screen shot twice: we can just copy the image file from the first one into the main folder. If we're fine with taking two different screenshots, that's fine, and it doesn't need to be a discussion item.